src/lib/connectors/shared/remote-debugging-connector.ts

```diff
@@ -597,7 +597,24 @@ export class Connector implements IConnector {
             }
 
             this._client = client;
-            const { Page } = client;
+            const { Page, Security } = client;
+
+            // bypassing the "Your connection is not private" certificate error when using self signed certificate in tests
+            // https://github.com/cyrus-and/chrome-remote-interface/wiki/Bypass-certificate-errors-(%22Your-connection-is-not-private%22)
+            // ignore all the certificate errors
+            if (this._options.overrideInvalidCert) {
+                Security.certificateError(({ eventId }) => {
+                    Security.handleCertificateError({
+                        action: 'continue',
+                        eventId
+                    });
+                });
+
+                await Security.enable();
+
+                // enable the override
```

Comment:
@qzhou1607 For consistency, start comment with uppercase letter and add `.` at the end (e.g. in this case: `Enable the override.`)

Note: I'll take care of these change when I merge this.